addons/binding/org.openhab.binding.bigassfan/src/main/java/org/openhab/binding/bigassfan/internal/BigAssFanHandlerFactory.java

```diff
@@ -23,6 +26,13 @@
  * @author Mark Hilbush - Initial contribution
  */
 public class BigAssFanHandlerFactory extends BaseThingHandlerFactory {
+    private Logger logger = LoggerFactory.getLogger(BigAssFanHandlerFactory.class);
+
+    @Override
+    protected void activate(ComponentContext componentContext) {
```

Comment:
Actually I mean if there is something more generic to get the binding version for the end-users. I don't know whether we should do it like this or that we should just let the users depend on Karaf. But let that be up to @kaikreuzer, @JueBag solution is already a step forward.